utils/screenshot.js

```diff
@@ -19,9 +22,17 @@ const options = {
   }
 }
 
+const generateFilePath = () => {
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir)
+  }
+
+  return `${dir}/${Date.now()}${Math.random()}.png`
+}
+
 const screenshot = async (url, git) => {
   const option = options[git]
-  let filePath = `tmp/${Date.now()}${Math.random()}.png`
+  let filePath = generateFilePath()
```

Comment:
Kenapa tidak menggunakan const ?

Reply:
because filePath will be set to null if page is not equal to url :smiley: